templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec puma -C ./config/puma.rb
```

Comment:
Right, @schneems  - as it is now, in this PR, neither the Procfile or puma.rb set the port. I should have been more specific.

@gabebw, I'm with you. I set it in the Procfile and all is good.